### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by jrtechcomm**
Assignment 7 ready for review by Laura. Unsure of where to add link to portfolio, as portfolio is not ready to show yet. GitHub projects will also be added in the future.

### CODE REVIEW

File: student-work/2022/Jason-Rosenberg/readme.md [hunk 1]
```diff
@@ -1 +1,47 @@
+**JASON CHAIM ROSENBERG**
```
Comment (Laura-Novich-OBW) [inline student-work/2022/Jason-Rosenberg/readme.md:1]:
```suggestion
# JASON CHAIM ROSENBERG
```

Comment (Laura-Novich-OBW) [reply student-work/2022/Jason-Rosenberg/readme.md:1]: This should be H1

File: student-work/2022/Jason-Rosenberg/readme.md [hunk 2]
```diff
@@ -1 +1,47 @@
...
 
+Central District, Israel | <comparc@icloud.com> | 053-336-7431
+# **Skills Summary**
```
Comment (Laura-Novich-OBW) [inline student-work/2022/Jason-Rosenberg/readme.md:4]:
```suggestion
# Skills Summary
```

Comment (Laura-Novich-OBW) [reply student-work/2022/Jason-Rosenberg/readme.md:4]: No need for bold as it is included in the style.

Comment (Laura-Novich-OBW) [reply student-work/2022/Jason-Rosenberg/readme.md:4]: Change the remaining headings, remove the ** asterisks.

File: student-work/2022/Jason-Rosenberg/readme.md [hunk 3]
```diff
@@ -1 +1,47 @@
...
+## ***Front-End Web Development, Hack Reactor, San Francisco, 2017***
+Short evening course in HTML5, CSS, and JavaScript.
+
+# **Experience**
+## ***Caliva.com, San Francisco***
+## ***Mission Organic, San Francisco***
+E-commerce Ops, 2018-2021
+
+Maintained B2C website, set up Google AdWords, and continually updated product offerings. Set up online business profiles. Performed various customer service tasks. Ensured compliance with California Bureau of Cannabis Control regulations.
+##
```
Comment (Laura-Novich-OBW) [inline student-work/2022/Jason-Rosenberg/readme.md:21]:
```suggestion
```

File: student-work/2022/Jason-Rosenberg/readme.md [hunk 4]
```diff
@@ -1 +1,47 @@
...
+##
+## ***Uber, San Francisco***
+### *Junior QA, 2013-2018*
+Beta tested driver and passenger apps, including driving, riding as a passenger, writing reports, and meeting with software engineers on a regular basis.
+## ***Intellectual Capital, LP, Los Angeles and Camp Hill, AL, USA***
+### *Partner, Ghost Writer, Business, and Legal Liaison, 2006-2014*
+Edited US Patent applications for electrical engineering professor emeritus, Chair of Computer Architecture, University of Alabama, and Auburn University. Handled business and legal communication in writing, on the phone, and in person. Worked with large Intellectual Property Law Firms in Atlanta and Irvine to prosecute multiple U.S. Patent applications that were granted by the USPTO. Worked on IP enforcement case vs. Texas Instruments. Sold my share of the partnership in 2014 at a profit.
```
Comment (Laura-Novich-OBW) [inline student-work/2022/Jason-Rosenberg/readme.md:28]:
```suggestion
Edited US Patent applications for electrical engineering professor emeritus, Chair of Computer Architecture at the University of Alabama, and Auburn University. Handled business and legal communication in writing, on the phone, and in person. Worked with large Intellectual Property Law Firms in Atlanta and Irvine to prosecute multiple U.S. Patent applications that were granted by the USPTO. Worked on an IP enforcement case vs. Texas Instruments. Sold my share of the partnership in 2014 at a profit.
```

**Review by Laura-Novich-OBW — CHANGES_REQUESTED** [review]

**Comment by Laura-Novich-OBW** [comment]
Some changes need to be made as per comments within. 
Make the changes and resubmit. 
You're almost there!

**Review by jrtechcomm — COMMENTED** [review]
Please re-read and approve. One change may not have worked.

**Pull request merged**